x/auth/module.go

```diff
@@ -86,7 +85,7 @@ func (AppModuleBasic) RegisterInterfaces(registry codectypes.InterfaceRegistry)
 
 // GetTxCmd returns the root tx command for the auth module.
```

Comment:
This is optional, can you remove the whole `GetTxCmd`?

Reply:
> This is optional, can you remove the whole `GetTxCmd`?

Done😀